chromium_src/ui/views/controls/focus_ring.cc

```diff
@@ -1,74 +0,0 @@
-// Copyright (c) 2019 The Brave Authors. All rights reserved.
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0. If a copy of the MPL was not distributed with this file,
-// you can obtain one at http://mozilla.org/MPL/2.0/.
-
-#include "base/check.h"
-#include "ui/color/color_id.h"
-#include "ui/color/color_provider.h"
-#include "ui/gfx/color_palette.h"
-#include "ui/native_theme/native_theme.h"
-#include "ui/views/widget/widget.h"
-
-// Override the Focus Ring's color.
-// In Chromium, this is specified via platform-specfic native theme,
-// using kColorId_FocusedBorderColor. However, only macOS Light native theme
-// overrides this. Since we do not have a Brave version of either
-// platform-specific, or common versions, and we only want to override a single
-// color, we use this micro-theme for the FocusRingView.
-namespace {
-
-ui::ColorId ColorIdForValidity(bool valid) {
-  return valid ? ui::kColorFocusableBorderFocused : ui::kColorAlertHighSeverity;
-}
-
-class FocusRingTheme {
- public:
-  SkColor GetSystemColor(::ui::ColorId color_id, bool is_dark) {
-    // At the time of implementation, only two Color IDs were possible.
-    // If this changes, consider overriding NativeTheme, or moving to
-    // ThemeProperties.
-    DCHECK(color_id == ui::kColorFocusableBorderFocused ||
-           color_id == ui::kColorAlertHighSeverity);
-    // Must be colors that are OK on dark or light bg since this is
-    // a very simplistic implementation.
-    switch (color_id) {
-      case ui::kColorFocusableBorderFocused:
-        return is_dark ? SkColorSetARGB(0x66, 0x3F, 0x39, 0xE8)
-                       : SkColorSetARGB(0x99, 0x3F, 0x39, 0xE8);
-      case ui::kColorAlertHighSeverity:
-        return SkColorSetRGB(0xf4, 0x34, 0x05);
```

Comment:
Not sure which color is set for invalid control's focused border color but I think deleting this doesn't make much difference.

New one:
<img width="936" height="152" alt="image" src="https://github.com/user-attachments/assets/d3c39175-394f-4a61-aed9-89b9d84ccb8e" />

existing one:
<img width="898" height="142" alt="image" src="https://github.com/user-attachments/assets/5a0c9fae-d34f-4106-a40f-375f39f37dfe" />